app/src/features/surveys/view/SurveyHeader.tsx

```diff
@@ -279,14 +240,6 @@ const SurveyHeader: React.FC<ISurveyHeaderProps> = (props) => {
               </Box>
             </Box>
             <Box ml={4} mb={4}>
-              <Button
-                variant="outlined"
-                color="primary"
-                className={classes.actionButton}
-                data-testid="publish-survey-button"
-                onClick={() => togglePublishSurvey()}>
-                {surveyWithDetails.surveyData.survey_details.publish_date ? 'Unpublish Survey' : 'Publish Survey'}
-              </Button>
```

Comment:
(not this line) On line 294 below, the delete tooltip still references not being able to delete a published survey.

Reply:
Can probably just drop the tooltip altogether (we dont have one for the project delete button)